frontend/src/App.js

```diff
@@ -1,24 +1,43 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { Routes, Route } from "react-router-dom";
+import Home from "./pages/Home";
+import Navbar from "./components/Navbar";
+import NotFound from "./components/NotFound";
+import Sidebar from "./components/Sidebar";
+import { useLocationChange } from "./hooks/useLocationChange";
+import Container from "./styles/Container";
+import History from "./pages/History";
+import Library from "./pages/Library";
+import YourVideos from "./pages/YourVideos";
+import LikedVideos from "./pages/LikedVideos";
+import Playlist from "./pages/Playlist";
+import Trending from "./pages/Trending";
```

Comment:
Make an `index.js` file inside the component folder and one inside the pages folder and import and export all the components and pages inside that file and use that respective index.js file to import components from a single file.